doc/make.bat

```diff
@@ -10,8 +10,9 @@ if "%SPHINXBUILD%" == "" (
 set SOURCEDIR=.
 set BUILDDIR=build
 set SPHINXPROJ=matplotlib
+if defined SPHINXOPTS goto skipopts
 set SPHINXOPTS=-W
```

Comment:
if we'
```suggestion
set SPHINXOPTS=-W --keep-building
```
that's what the docs say is the default and that's what's set in the makefile